components/QueueTable.js

```diff
@@ -35,6 +35,7 @@ const ShowUploadQueue = (props) => {
       marginTop: "20px",
       width: "100%",
       zIndex: 0,
+      cursor: "pointer",
```

Comment:
The `cursor: "pointer"` should only come when I hover over the link in Job ID and Title. Currently, it's coming anywhere I hover on the table.

Reply:
To make only the job ID and Title cursor's pointer, I will need to add an empty href to both of those links like, ```
<a sx={styles.id} href="#" onClick={() => onClick(value)}>
          {value}
        </a>
```, as I could not find any other way to make only those two as pointers. Then after click on any item it will change the url to `<url>/queue/#` ,  is it okay to make this change?